deploy/job-manager/job-manager-deployment.yaml.ctmpl

```diff
@@ -22,22 +22,28 @@ spec:
         ports:
```

Comment:
Add `imagePullPolicy: Always` below the image to prevent GKE from evilly(sometimes) remembering the old images.